### DISCUSSION TITLE
【开源自荐】🚀一款轻量级云原生多数据源监控告警引擎（运维研发必备能力）

### DISCUSSION THREAD

**Discussion by Cairry** [just-talks/tech-talks#74] [3.分享创造]
## 💎 项目介绍
WatchAlert 是一款为云原生环境量身打造的轻量级监控告警系统，专注于**可观测稳定性**主题，提供全面的监控与告警支持。

**能力**
- Metrics 监控
    - 集成：Prometheus、VictoriaMetrics
- Logs 监控
    - 集成：Loki、ElasticSearch、阿里云日志服务 (AliCloud SLS)
- Traces 监控
    - 集成：Jaeger
- Events 监控
    - 集成：Kubernetes
- Network 监控
    - 集成：HTTP、ICMP、TCP、SSL
- 告警通知
  - 飞书、钉钉、企业微信、邮件、自定义Hook

**为什么选择 WatchAlert？**
- 针对云原生环境优化，轻量易用。
- 灵活的告警规则配置，支持多种数据源。
- 专注于稳定性场景，助力快速发现与解决问题。
- 提供稳定可靠的监控告警解决方案，助力用户提升运维效率，降低维护成本。

## 📄 项目文档
- GitHub：https://github.com/opsre/WatchAlert
- 功能介绍：https://cairry.github.io/docs/
- 快速部署：https://cairry.github.io/docs/install/docker.html

**Comment by kalxd** [comment]
不错，支持一个。

**Reply by Cairry** [reply]
🫶 thanks, 冲！
